src/Aspire.Hosting.AWS/Aspire.Hosting.AWS.csproj

```diff
@@ -8,7 +8,7 @@
     <PackageTags>aspire hosting aws</PackageTags>
     <Description>Add support for provisioning AWS application resources and configuring the AWS SDK for .NET.</Description>
     <!-- This package hasn't shipped stable, so package validation runs against previously shipped version of it. -->
-    <PackageValidationBaselineVersion>8.0.1-preview.8.24267.1</PackageValidationBaselineVersion>
+    <PackageValidationBaselineVersion>8.1.0-preview.1.24373.2</PackageValidationBaselineVersion>
```

Comment:
I don't want to block the rest of this PR, but I think we shouldn't do package validation until we actually ship stable. (Or at least, don't do public API validation in the package validation)

The reasoning is that we are free to change the public APIs until the library ships stable. Doing package validation just gets in the way and confuses people when they try to change the APIs. For example, in https://github.com/dotnet/aspire/pull/4950/, I needed to disable validation because I was updating to the new Azure.AI.OpenAI dependency.

Reply:
> I needed to disable validation because I was updating to the new Azure.AI.OpenAI dependency.

Instead of disabling validation, you can alternatively just pass the property `/p:GenerateCompatibilitySuppressionFile=true` as indicated [here](https://learn.microsoft.com/en-us/dotnet/fundamentals/apicompat/diagnostic-ids#suppression-file) in order to generate a suppression file.